python/tvm/relay/def_use.py

```diff
@@ -0,0 +1,42 @@
+from . import expr
+from .expr_functor import ExprVisitor
+from .analysis import free_vars
+
+import attr
+from typing import List, Dict
+
+@attr.s(auto_attribs=True)
+class DefUse:
+    defn: expr.Var
+    uses: List[expr.Expr]
+
+class DefUseAnalysis(ExprVisitor):
```

Comment:
not in use right now?

Reply:
This was in prep for the next piece I can remove from this PR.